src/web-vitals-autoinstrumentation.ts

```diff
@@ -45,6 +56,116 @@ interface VitalOpts extends ReportOpts {
   applyCustomAttributes: ApplyCustomAttributesFn;
 }
 
+// To avoid importing InstrumentationAbstract from:
+// import { InstrumentationAbstract } from '@opentelemetry/instrumentation/build/src/instrumentation';
+// When this is exposed we can import from there.
+export abstract class InstrumentationAbstract implements Instrumentation {
```

Comment:
My only $0.02 about importing this vs copying it in is that I'm worried about code drift from the base package but I'll leave it up to you! I definitely see advantages of doing it this way but wanted to mention it.

Reply:
:100: We'll be sure to follow up if/when that base class is exposed, but in the meantime, we're at least inline with the what the library is publicly exposing which feels marginally safer.